src/ducks/Watchlists/Actions/Edit/EditForm/hooks.js

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useMemo } from 'react'
+import gql from 'graphql-tag'
+import { useQuery } from '@apollo/react-hooks'
+
+const ALL_PROJECTS = gql`
+  {
+    allProjects {
+      id
+      name
+      ticker
+      logoUrl
+      darkLogoUrl
+    }
+  }
+`
+
+export const useAllProjects = filter => {
+  const { data, loading, error } = useQuery(ALL_PROJECTS)
```

Comment:
Reuse [this hook](https://github.com/santiment/app/blob/04bdce9ef63c55e03f55c92f4cdebea6a2d56baa/src/stores/projects/index.js#L29) for fetching projects since it's called on app boot and the data is cached